### PR TITLE
Feat: Added Badge

### LINKED ISSUE #14: Update : badge.html
adding sidebar contents in badge.html

### PR DESCRIPTION
**Pull request by RahulSingh9131**
1. badge on icons
IconBadge
2. badge on avatars
avatarBadge
3. badge on buttons
buttonBadge

### CODE REVIEW

File: css/components/badge.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    height: 1.1rem;
+    border: .1rem solid #fff;
+    border-radius: 50%;
+    color: #fff;
+    background-color: var(--primary-dark);
+}
+
+/*== css for badges on avatars == */
+.badge-online{
+    position: absolute;
```
Comment (vivekmehto) [inline css/components/badge.css:27]: Avoid repeating code.

File: css/components/badge.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    height: .9rem;
+    width: .9rem;
+    background-color: var(--danger);
+    border: .1rem solid #fff;
+    border-radius: 50%;
+}
+
+/*== css for badges on button ==*/
+.btn-badge{
+    font-size: var(--normal-font-size);
```
Comment (vivekmehto) [inline css/components/badge.css]: exactly the same properties with the different class names btn-badge and badge.

**Review by vivekmehto — COMMENTED** [review]
Avoid repeating code as much as you can. use different classes for changing properties.
you have repeated same 4-5 lines in each class

File: docs/avatar/avatar.html [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+            <header>
+                <div class="container">
+                    <div class="logo">
+                        <img src="../../assets/brand-logo.svg" alt="Logo">
+                        <h3>Pegasus</h3>
+                    </div>
+                    <nav class="links">
+                        <ul>
+                           <li><a href="https://github.com/RahulSingh9131">Github</a></li>
+                           <li><a href="https://twitter.com/singhrahul3112">Twitter</a></li>
```
Comment (shubhamkumar648) [inline docs/avatar/avatar.html]: you could add _blank attribute for open in new tab

**Review by shubhamkumar648 — COMMENTED** [review]
looks good ! use comment in html file for easy readble

File: docs/avatar/avatar.html (hunk 3 above)
Comment (RahulSingh9131) [reply docs/avatar/avatar.html]: sure will add that attribute.

File: css/components/badge.css (hunk 1 above)
Comment (RahulSingh9131) [reply css/components/badge.css:27]: ok will do that.

File: css/components/badge.css (hunk 2 above)
Comment (RahulSingh9131) [reply css/components/badge.css]: sure will change that

**Pull request merged**